docs/configuration-reference.md

```diff
@@ -2252,6 +2252,8 @@ consul_exporter: <consul_exporter_config>
 # Controls the windows_exporter integration
 windows_exporter: <windows_exporter_config>
 
+kafka_lag_exporter: <kafka_lag_exporter_config>
```

Comment:
Hmm, I'd prefer keeping the name of the original exporter (`kafka_exporter`) intact here. Is there a specific reason we'd want to name it differently?